lib/src/model.dart

```diff
@@ -34,11 +31,11 @@ class Model {
       // transform classes into model descriptions and loop through them
       modelDefinitions.forEach((m) {
         // start entity
-        var entityUtf8 = Utf8.toUtf8(m["entity"]["name"]);
+        var entityUtf8 = Utf8.toUtf8(m["name"]);
```

Comment:
Seeing this as well as rest of the code, I have a feeling it would be much cleaner with a "normal" class-based representation of a model. Nothing fancy is necessary, just the fields that are currently present in the json "entities" path

Reply:
Done.